Neos.ContentRepositoryRegistry/Classes/Factory/PrivilegeProvider/FakePrivilegeProvider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+declare(strict_types=1);
+namespace Neos\ContentRepositoryRegistry\Factory\PrivilegeProvider;
+
+use Neos\ContentRepository\Core\Factory\ContentRepositoryId;
+use Neos\ContentRepository\Core\Projection\ContentGraph\VisibilityConstraints;
+use Neos\ContentRepository\Core\SharedModel\Privilege\ContentStreamPrivilege;
+use Neos\ContentRepository\Core\SharedModel\Privilege\PrivilegeProviderInterface;
+use Neos\ContentRepository\Core\SharedModel\Privilege\Privileges;
+use Neos\ContentRepository\Core\SharedModel\User\UserIdProviderInterface;
+use Neos\ContentRepository\Core\SharedModel\Workspace\ContentStreamIds;
+use Neos\ContentRepositoryRegistry\ContentRepositoryRegistry;
+
+/**
+ * @internal
+ */
+final class FakePrivilegeProvider implements PrivilegeProviderInterface
```

Comment:
Not really a `FakePrivilegeProvider` but a first implementation to test content stream access